b2.js

```diff
@@ -290,6 +295,19 @@
             break;
         }
 
+        if (valueDataType) {
+          switch (valueDataType.toLowerCase()) {
+            case 'int':
+              fieldValue = isNaN(fieldValue) ? fieldValue : parseInt(fieldValue);
```

Comment:
If the filedValue is empty string, the isNaN will return false, but parseInt will return NaN, and you will get NaN value at last.